assets/tests/hooks/useDetour.test.ts

```diff
@@ -363,13 +385,13 @@ describe("useDetour", () => {
     act(() => result.current.addConnectionPoint?.({ lat: 0, lon: 0 }))
 
     await waitFor(() => {
-      expect(result.current.missedStops).not.toBeUndefined()
+      expect(result.current.missedStops).not.toStrictEqual([])
```

Comment:
I'm surprised the linter didn't catch this, can we use `toHaveLength(0)` instead? https://jestjs.io/docs/next/expect#tohavelengthnumber

(same for a few other instances of this)

Reply:
Ah yeah - I wanted to use `toBeEmpty()`, but for some reason that claimed not to be a function, even thought Typescript was happy with it.

Will fix.

Reply:
[Done](https://github.com/mbta/skate/pull/2486/commits/ae850c1973c49100f4c057e6c1474e17263bc806)

Reply:
> Ah yeah - I wanted to use `toBeEmpty()`, but for some reason that claimed not to be a function, even thought Typescript was happy with it.

Ah, yeah, that's a testing-library jest-dom matcher for DOM stuff, unfortunately sounds very similar to what you'd expect to use for empty arrays(at least it's been deprecated). You were probably getting the error because _likely_ (I didn't check) you don't have the jest-dom matchers imported.

https://github.com/testing-library/jest-dom?tab=readme-ov-file#tobeempty